src/In.ProjectEKA.HipService/Gateway/GatewayClient.cs

```diff
@@ -0,0 +1,46 @@
+namespace In.ProjectEKA.HipService.Gateway
+{
+    using System;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using Common;
+    using Logger;
+    using static Common.HttpRequestHelper;
+
+    public class GatewayClient
```

Comment:
Wasn't GatewayClient introduced as part of Discovery API?

Reply:
yes it is just moved in different package